scripts/deploy-upgradable.js

```diff
@@ -103,7 +103,7 @@ async function deploy(env, chains, wallet, artifactPath, contractName, deployTo)
                 implementationJson,
                 args,
                 getUpgradeArgs(contractName, chain),
-                chain.gasOptions,
+                _.get('gasOptions.gasLimit', chain),
```

Comment:
Change the import to be `const { get } = require('lodash/fp');` so that here it becomes `get('gasOptions.gasLimit', chain),`

Reply:
yes

Reply:
sounds good, just updated